lib/valkyrie/persistence/active_fedora/orm/resource.rb

```diff
@@ -39,6 +39,8 @@ def initialize(uri = RDF::Node.new, _parent = ActiveTriples::Resource.new)
     property :edit_groups, predicate: ::RDF::URI("http://example.com/edit_groups")
     property :edit_users, predicate: ::RDF::URI("http://example.com/edit_users")
   end
+
+  # A PCDM work object used for serializing all Valkyrie::Models to ActiveFedora
```

Comment:
Can you provide an `@see` that provides a URL to what PCDM work object you mean?